tekton_configs/pytest-task.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: pytest
+  namespace: gabrielwen
+  annotations:
+    sidecar.istio.io/inject: "false"
+spec:
+  inputs:
+    params:
+    - name: test-worker-img
+      type: string
+      description: Test worker image.
+      default: gcr.io/kubeflow-ci/test-worker:latest
+    - name: src-dir
```

Comment:
Why is src-dir a parameter? My expectation would be that all source we provided using tekton resources corresponding to git resources. So you you would then reference the location as 
/workspace/$(inputs.resources.<repo name>.name)

Reply:
git resource is not able to change at runtime.  it's a PipelineResource that is similar to Task - we need to deploy with `kubectl apply` first.